### ISSUE TITLE
[UI/SkeletonLoaderAnimation] Animation is not smooth on WASM

### ISSUE THREAD

**Issue by Marc-Antoine-Soucy** [unoplatform/Uno.Samples#490]
![jittery](https://github.com/unoplatform/Uno.Samples/assets/90481654/2f9fcbc8-d967-4584-8e73-d6bbfbf16a7f)

**Comment by nickrandolph** [comment]
@jeromelaban if the animations are running slow, is this something we want to raise on uno repo?

**Comment by jeromelaban** [comment]
Yes, though it may be related to the interpreter and/or the version of the bootstrapper. Enabling AOT or updating the bootstrapper is likely to help.
